### DISCUSSION TITLE
sentry/sentry-laravel incorrect handling

### DISCUSSION THREAD

**Discussion by belamov** [composer-unused/composer-unused#314] [Q&A] [answered]
hello, thanks for the tool, it is very useful

i have a problem, but it also may be a bug - it is connected with zombies

in my [project](https://github.com/RehearsalsBooking/backend) i am usung [sentry/sentry-laravel](https://github.com/getsentry/sentry-laravel) package

it is not used in my code directly, this is just some kind of glue for two projects, so i am excluding it when i run a command

on version 0.7 everything worked as expected, this package was ignored and no error occured

but on version 0.8 when zombies were added, command now failing - it detects this package as zombie, but I know for sure that I am using this package, just indirrectly

if i remove this package from ignored - then command detects this package as unused - so no matter what it doesn't allow me to handle this package for my checks

here is my source code https://github.com/RehearsalsBooking/backend - it would be great if someone would help me to figure out why is this happening

**Comment by icanhazstring** [comment, marked as answer]
Hi @belamov 

This might be already fixed in `dev-main` which will be released soon :)
Just try and quickly install the dev-main package to your repo and try it out.

I just tested it using PHP8.1 and latest commit. Ignoring the package using `--excludePackage=sentry/sentry-laravel` or adding it as ignored into the new `composer-unused.php` config worked.

**Reply by belamov** [reply]
yes, yua are right, error doesnt occur at `dev-main`
thanks! will wait for release!
